src/components/Avatar.tsx

```diff
@@ -48,12 +48,12 @@ type AvatarProps = {
 };
 
 function Avatar({
-    source,
-    imageStyles,
-    iconAdditionalStyles,
-    containerStyles,
+    source = undefined,
```

Comment:
Does this rule enforce setting a default values for every optional prop? That's a bummer imo 😒 cc @ishpaul777 @roryabraham

Reply:
That is something we would like to avoid with Typescript, that's why this rule wasn't applied to Ts files before, wdyt @fabioh8010?

Reply:
Yes, is unnecessary to set `undefined` for every optional prop without default values because both JS and TS will already consider that the variable can be `undefined`, let's not enforce this 🙂